packages/lesswrong/lib/collections/users/custom_fields.ts

```diff
@@ -693,6 +693,10 @@ addFieldsDict(Users, {
     hidden: !hasEventsSetting.get(),
     ...notificationTypeSettingsField({ channel: "both" }),
   },
+  notificationPostsNominatedReview: {
```

Comment:
You want to also add this to the `UsersCurrent` fragment

Reply:
UsersCurrent or UsersEdit? Edit seems to be where the existing settings live